run_sdk_container

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+#
+# Copyright (c) 2021 The Flatcar Maintainers.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -eu
+
+cd $(dirname "$0")
+source sdk_lib/sdk_container_common.sh
+
+version="$(get_version_from_versionfile)"
+arch="all"
+checkout="true"
+create_only="false"
+name=""
+
+usage() {
+    echo "  $0 - Start an SDK container of a given SDK release version."
+    echo "       This will create the container if it does not exist, otherwise it will just start it."
+    echo "       If the container is already running then it will exec into the container."
```

Comment:
Having to start and stop the container will make it a bit complicated to use in scripts and using a fixed container name even forbids concurrent usage. I think this needs to be reworked to be useful in the build pipeline with the goals for easy automatic cleanup of containers and concurrent usage.

Reply:
That's a good point - the script is biased towards interactive use. I'll add a `--oneshot` flag to make it better suited for scripting use.

Reply:
The Jenkins scripts make use of entering the SDK multiple times. Maybe we can simplify this but it would be nice to keep the behavior of being able to exec into the container but using it's temporary name (use the one allocated by the container runtime?). E.g., have the container being started with `docker run --rm` in the background so that killing the process will also remove the container (we already have a way of killing a job's processes in Jenkins). By saving the temporary container name to a local file we could know how to exec into it on the next run.

Reply:
We can provide a custom name via `-n` and make sure that's random. However, I feel Jenkins should not re-use containers but instead produce container images to then be re-used... feels cleaner.
Thinking about it, we might even add yet another Dockerfile for building binary packages and then publish a super-fat SDK which ships all the board packages precompiled.
Thanks for bringing up the `--rm` flag, that was missing from my (ongoing) oneshot implementation!

Reply:
The Jenkins jobs need to run set up an SDK container (=`cork create`) and then execute some commands in it (=`cork enter`). You already have this implemented here which is ok, but it needs to be improved a bit.

Reply:
Killing the docker client doesn't actually remove the container with `run --rm`, something else is needed…

Reply:
When there are Docker containers left over when canceling a Jenkins job, the build process in the Jenkins job will continue to write to the folder which can corrupt the state of the next job depending on how the folders are reused (and wastes RAM/CPU resources).

Reply:
My suggestion is that we start a background process `sleep infinity &`, remember the PID, and start the container with `docker run --rm -v /proc/$PID:/watchdir sh -c 'while ls /watchdir > /dev/null; do sleep 5; done'`
This will ensure that the container exits when the `sleep` process is killed.

Reply:
Implemented + tested in Jenkins.